### PR TITLE
bubble_sort

### PR DESCRIPTION
**Pull request by emmyobonyo**
In this project we
-Used ruby to implement **bubble_sort** and **bubble_sort_by** algorithms.

### CODE REVIEW

**Review by ElijahTobs — COMMENTED** [review]
# Status: 🚨 ❌ INVALID ❌ 🚨 

## Hi @emmyobonyo and @tmampa  :wave: 

This project has been marked as invalid because of the following:
* [ ] The project should have a descriptive README.md. To be descriptive the README needs simply to include at least the title of the project, a description about it, and instructions on how to setup and run the project. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [ ] Pull Request should have a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))



### Make the required changes and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by mke2111 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED 🔴 ♻️ 

## Hey Happy Coders, 👋🏽 
**This is a great project you have going on, well-done lads and thanks for implementing the changes requested earlier on**. 

### Well-done 💯 
    1. The algorithm runs just fine and so do the linter checks.
  

**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

### REQUIRED

- [x] The getting started instructions are not very clear. Write step-by-step instructions to follow to get a user to run/test their input against your algorithm.
- [x] The link you have [provided for the cloning](https://github.com/ZeenatLawal/bubble-sort) process directs to another repo that is not yours.  Let's be careful with what we copy/paste in our projects.

#

### Kindly make all the requested changes and submit another CRR. 💯 

**Cheers 🍹 and Happy coding 💻** 

<hr>

Mention @mke2111 in this thread and I'll be glad to help. 😄 
### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)

File: README.md [hunk 1]
```diff
@@ -1,65 +1,46 @@
...
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+- [Copy this link](https://github.com/ZeenatLawal/bubble-sort)
+- Open your terminal or command line
+- Run "git clone Paste [this link](https://github.com/ZeenatLawal/bubble-sort)"
```
Comment (mke2111) [inline README.md:20]: This link doesn't seem to be the correct link to the repo. Change the link.

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @emmyobonyo and @tmampa 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed.

##

Kindly work on the suggested changes  👇.
- [ ] You can replace Run `git clone Paste [this link](https://github.com/emmyobonyo/bubble-sort-algot)` with `git clone https://github.com/emmyobonyo/bubble-sort-algo.git` it will be more meaningful and you can remove the first point with updating this.

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad (Code Reviewer)](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
- All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.
- Congratulations on the splendid job @emmyobonyo ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
HAPPY CODING :computer:
**[Umair Arshad (Code Reviewer)](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**